site/content/docs/2.0/components/callouts.md

```diff
@@ -21,5 +21,11 @@ Callouts are similar to alerts, in that they call attention to a small portion o
   <p>A callout is useful for drawing attention to an important piece of content. You can use any of the available brand and theme colors in conjunction with the callout to change its border and text color.</p>
 </div>
 {{ end }}
+{{ range (index $.Site.Data "theme-colors") }}
+<div class="callout callout-{{ .name }}{{ if eq .name "white" }} bg-dark{{ end }}">
+  <p class="h4">{{ .name | title }} Callout</p>
```

Comment:
This would match the styles more closely in the 5.0 version that has the following CSS.
```
.callout .h4, .callout .text-size-h4, .callout h4 {
    margin-top: 0;
    margin-bottom: 0.25rem;
}
```

Reply:
I believe we would want it to be mt-0?

```suggestion
  <p class="h4 mt-0">{{ .name | title }} Callout</p>
```